mypy/version.py

```diff
@@ -7,13 +7,11 @@
 # - For 1.0 we'll switch back to 1.2.3 form.
 __version__ = '2.0.0+dev'
 base_version = __version__
-mypy_version = "0.940+dev"
+mypy_version = "0.940+dev.4602c4dad026dcc64cb94d531d5ce4615c7c3a01"
```

Comment:
is there anything we can implement to prevent peopel from forgetting to update this when pulling from upstream?

Reply:
I've changed it to not have the hash at all in dev installs

Reply:
ik, i mean when you pull in changes from mypy you gotta remember to manually update this... ig we could make a script to do it or something in the ci that somehow checks that the hash is correct, idk

Reply:
sorry, I've pushed it now.